pkg/config/apm.go

```diff
@@ -112,6 +112,12 @@ func setupAPM(config Config) {
 	config.BindEnv("apm_config.telemetry.additional_endpoints", "DD_APM_TELEMETRY_ADDITIONAL_ENDPOINTS")
 	config.BindEnv("apm_config.obfuscation.credit_cards.enabled", "DD_APM_OBFUSCATION_CREDIT_CARDS_ENABLED")
 	config.BindEnv("apm_config.obfuscation.credit_cards.luhn", "DD_APM_OBFUSCATION_CREDIT_CARDS_LUHN")
+	config.BindEnv("apm_config.obfuscation.mongodb", "DD_APM_OBFUSCATION_MONGODB")
+	config.BindEnv("apm_config.obfuscation.elasticsearch", "DD_APM_OBFUSCATION_ELASTICSEARCH")
+	config.BindEnv("apm_config.obfuscation.redis.enabled", "DD_APM_OBFUSCATION_REDIS_ENABLED")
+	config.BindEnv("apm_config.obfuscation.memcached.enabled", "DD_APM_OBFUSCATION_MEMCACHED_ENABLED")
+	config.BindEnv("apm_config.obfuscation.http", "DD_APM_OBFUSCATION_HTTP")
+	config.BindEnv("apm_config.obfuscation.remove_stack_traces", "DD_APM_OBFUSCATION_REMOVE_STACK_TRACES")
```

Comment:
These configs are already defined up in the same function but with `SetKnown`, not with `BindEnv`.
Instead of redefining them, we should rather update the existing definitions [here](https://github.com/DataDog/datadog-agent/blob/be63ac38c088c6eb7b11834cf3202dfef95e9e0f/pkg/config/apm.go#L23-L39) and use `BindEnv` instead of `SetKnown`.

A part from that, let's make sure we cover all the [obfuscation options](https://github.com/DataDog/datadog-agent/blob/be63ac38c088c6eb7b11834cf3202dfef95e9e0f/pkg/config/apm.go#L23-L39) we have, i.e we should have an env var for every single obfuscation option.